content/en/containers/kubernetes/configuration.md

```diff
@@ -11,18 +11,41 @@ aliases:
 
 After you have installed the Datadog Agent in your Kubernetes environment, you may choose additional configuration options.
 
-## Live Containers
+### Enable Datadog to collect:
+- [Traces (APM)](#enable-apm-and-tracing)
+- [Kubernetes events](#enable-kubernetes-event-collection)
+- [NPM](#enable-npm)
+- [Logs](#enable-log-collection)
+- [Processes](#enable-process-collection)
 
-The [Datadog Agent][1] and [Cluster Agent][2] can be configured to retrieve Kubernetes resources for [Live Containers][3]. This feature allows you to monitor the state of pods, deployments and other Kubernetes concepts in a specific namespace or availability zone, view resource specifications for failed pods within a deployment, correlate node activity with related logs, and more.
+### Other capabilities
+- [Datadog Cluster Agent](#datadog-cluster-agent)
+- [Integrations](#integrations)
+- [Containers view](#containers-view)
+- [Orchestrator Explorer](#orchestrator-explorer)
+- [External metrics server](#external-metrics-server)
```

Comment:
```suggestion
- [External metrics server](#custom-metrics-server)
```
(the sub-title link goes to "custom metrics server" on the page, didn't know if you also want to change the name here from "External" to "Custom" as well ?